src/node/chainstatemanager_args.cpp

```diff
@@ -60,8 +60,7 @@ util::Result<void> ApplyArgsManOptions(const ArgsManager& args, ChainstateManage
         script_threads += GetNumCores();
```

Comment:
`// -par=-n means "leave n cores free" (number of cores - n - 1 script threads)` is this still true?